default.cfg

```diff
@@ -43,6 +43,7 @@ music_volume		8
 parallelism_patch_columns		3
 parallelism_realcolor_tint		8
 scale_melt		true
+semi_translucent_fuzz	true

```

Comment:
`semi_translucent_fuzz` should be false by default since it's not vanilla. Is it the unique effect that Maes added to Mocha Doom and that was the default in 32-bit mode?